redirect.py

```diff
@@ -41,8 +41,20 @@ def get(self):
           self.response.write(json.dumps(result))
       elif self.request.get('site') == 'comments':
           urlfetch.set_default_fetch_deadline(10)
+          startIndex = self.request.get('start')
+
+          try:
+            startIndex = int(startIndex)
+          except:
+            startIndex = 0
+
+          maxResults = self.request.get('max')
+          try:
+            maxResults = int(maxResults)
+          except:
+            maxResults = 0
           self.response.headers.add_header("Access-Control-Allow-Origin", "*")
-          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId')).execute()
+          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId'), maxResults=max, startIndex=startIndex).execute()
```

Comment:
I have uploaded a new patch which limits us to 200 comments per pull. 

Also took a look at the 500-ing issue seems like I have an authorization issue. Its showing up as a 401 issue on my console. I get the same issue when I just access the bug through monorail directly. Here for example: https://monorail-prod.appspot.com/_ah/api/monorail/v1/projects/chromium/issues/668012/comments?maxResults=0&alt=json&startIndex=0

Is it possible that only you and dstockwell are authorized through chromez when youre building it locally?